students/kosarzewski_maciej/lesson_03_functions/exponentiation.py

```diff
@@ -0,0 +1,17 @@
+def power(a, n):
+    if a == 0:
+        return 0
+    elif n == 1:
+        return a
+    elif n == 0:
+        return 1
+    else:
+        return a * power(a, n - 1)
+
+
+print ("Compute a^n")
+print ("Give base: ")
+base = float(input())
```

Comment:
Please use input argument to print info about user input
float(input('Give base: '))